ctl/Cargo.toml

```diff
@@ -1,7 +1,7 @@
-[package]
-name = "ctl"
-version.workspace = true
-edition.workspace = true
-lints.workspace = true
-
-[dependencies]
+[package]

```

Comment:
Please [configure](https://stackoverflow.com/a/13154031/7445826) your Git to use LF instead of CRLF. Note that commits unnecessarily changed entire unrelated files, such as this one.